codec/src/main/java/io/netty/handler/codec/ReplayingDecoderByteBuf.java

```diff
@@ -89,6 +89,11 @@ public ByteBufAllocator alloc() {
     }
 
     @Override
+    public boolean isReadOnly() {
```

Comment:
not sure if this is semantically correct as it not throws "ReadOnlyBufferException" on write/set operations